app/views/SearchMessagesView/index.js

```diff
@@ -168,6 +173,27 @@ class SearchMessagesView extends React.Component {
 		}
 	}
 
+	onEndReached = async() => {
```

Comment:
Most of this function is duplicated from `this.search`.
Can we reuse it instead?